src/alpaca_typer.erl

```diff
@@ -2816,9 +2815,9 @@ typ_of(Env, Exp) ->
         {Typ, NewVar} -> {unwrap(Typ), update_counter(NewVar, Env)}
     end.
 
-from_code(C) ->
-    {ok, E} = alpaca_ast_gen:parse(alpaca_scanner:scan(C)),
-    E.
+%% from_code(C) ->
```

Comment:
I didn't knew where it was coming from, it's not called but I figured it might have some reason. I'm happy to take it out so ;)
there are two more of those lingering around.